act-rules-format.md

```diff
@@ -497,6 +501,66 @@ Background (optional) {#background}
 An ACT Rule <em class="rfc2119">may</em> contain information about the background for the development of the rule, or references to relevant reading. The background information is optional, but whenever it is included in the rule, the relationship to the relevant reading can be specified. Examples of relevant background references for a rule for a [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) [[WCAG]] success criterion could be [WCAG 2.1 Understanding documents](https://www.w3.org/WAI/WCAG21/Understanding/), [WCAG 2.1 Techniques](https://www.w3.org/WAI/WCAG21/Techniques/), or [WAI-ARIA 1.1](https://www.w3.org/TR/wai-aria/), CSS [[css-2018]], or HTML [[HTML]] specifications.
 
 
+Implementations (optional) {#implementation}
+--------------------------------
+
+An ACT Rule <em class="rfc2119">may</em> contain a list of implementations that are consistent with that rule. An [=implementation=] is an accessibility test methodology or test tool that can be used produce [=outcomes=] of an ACT Rule.
+
+An [=implementation=] consists of multiple [=implementation procedures=]. The [=Implementation procedures=] are not required to have a one-to-one mapping to an ACT Rule. Mapping of an implementation to an ACT Rule is done by determining the set of implementation procedures with the highest consistency and coverage of that rule. Multiple implementation procedures can combine to map to a single ACT Rule, or a single implementation procedure can map to multiple ACT rules simultaneously.
+
+For each implementation the following data <em class="rfc2119">may</em> be provided. If included, consistency and coverage <em class="rfc2119">must</em> be determined as defined in this section.
+
+- Name of implementation
+- Name of the vendor
+- [Consistency](#impl-consistency)
+- [Coverage](#impl-coverage) 
+- Version used in testing consistency and coverage
+
+### Consistency ### {#impl-consistency}
+
+Consistency says how well a set of [=implementation procedures=] are aligned with the intent of an ACT Rule. Consistency is determined by comparing the [=outcome=] that implementation procedures return on the test cases to the expected outcomes of the test cases. In determining consistency, the `passed` and `inapplicable` outcome are largely interchangeable. This supports implementations that do not distinguish between `passed` and `inapplicable`, and allows implementation procedures to be scoped different from an ACT Rule.
```

Comment:
```suggestion
Consistency says how well a set of [=implementation procedures=] are aligned with the intent of an ACT Rule. Consistency is determined by comparing the [=outcome=] that implementation procedures return on the test cases to the expected outcomes of the test cases. In determining consistency, the `passed` and `inapplicable` outcomes are largely interchangeable. This supports implementations that do not distinguish between `passed` and `inapplicable`, and allows implementation procedures to be scoped different from an ACT Rule.
```

Reply:
I'm debating if I like the sentence "In determining consistency, the `passed` and `inapplicable` outcomes are largely interchangeable". It begs the question when *aren't* they interchangeable and why.

Reply:
Hmm, yeah I guess they are interchangeable. I'll update that. The reason I put it there was for minimal, but there it's the inapplicable test cases that are treated different from passed, not the inapplicable outcomes.